src/entries/popup/pages/send/index.tsx

```diff
@@ -116,6 +134,11 @@ export function Send() {
     sortMethod,
   } = useSendAsset();
 
+  const filteredNonHiddenAssets = useMemo(
```

Comment:
Whatever name you choose for the above comment, i'd mirror that here since we're doing the same operation.